mep/DEVELOPERNOTES.rst

```diff
@@ -0,0 +1,11 @@
+
+
```

Comment:
nice idea to document this; please add a little bit of context - probably enough to add a title for the page and a subheading + brief description for this section